frontend/static/federation.json

```diff
@@ -119,7 +119,7 @@
       "i2p": ""
     },
     "testnet": {
-      "onion": "https://qu6xztmzhlve6nxbb77jldek53pvhkaltz6seni7wq6g6yyj233qp4yd.onion",
+      "onion": "https://testraliar7xkhos2gipv2k65obykofb4jqzl5l4danfryacifi4t7qd.onion",
```

Comment:
Nice vanity address :)

Let's make a rule starting after this PR to not edit `federation.json` on bug fix PRs unless it's a re-factorization of the JSON itself.